scripts/gen_istio_image_list.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+# 
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script generates the list of images used by Istio on the control
+# plane, data plane as well as addons.We generate this list by looking
+# at the values.yaml file where the Helm charts are stored, as well as
+# reading the values of DOCKER_TARGETS rule in the istio-docker.mk file.
+ 
+ISTIO_IMAGES_FILE=istio.IMAGES
+ISTIO_HELM_CHART_DIR="../install/kubernetes/helm/istio/charts/"
+
+function get_istio_images() {
+    buff=''
+    continue_processing=0
+    while IFS= read -r line 
+    do 
+       if [[ $line == DOCKER_TARGETS:=*\\ ]]
+       then
+           buff+=${line#DOCKER_TARGETS:=}
+           continue_processing=1
+       else
+           if [ $continue_processing -eq 1 ]
+           then
+              buff=${buff%\\}$line
+              break
+           fi
+       fi
+    done < ../tools/istio-docker.mk
```

Comment:
This should use absolute paths. See some other scripts where we do stuff to determine the WD and ROOT. Otherwise this fails like:
```
$ ./scripts/gen_istio_image_list.sh
./scripts/gen_istio_image_list.sh: line 25: ../tools/istio-docker.mk: No such file or directory
```

Reply:
I removed these lines since I changed the logic to get this info from release and not from master.